guides/source/active_record_postgresql.md

```diff
@@ -102,16 +100,20 @@ class Profile < ApplicationRecord
 end
 
 # Usage
-Profile.create(settings: { "color" => "blue", "resolution" => "800x600" })
+Profile.create(settings: { "color" => "blue",
+                           "resolution" => "800x600" })
 
 profile = Profile.first
 profile.settings # => {"color"=>"blue", "resolution"=>"800x600"}
 
-profile.settings = {"color" => "yellow", "resolution" => "1280x1024"}
+profile.settings = {"color" => "yellow",
+                    "resolution" => "1280x1024"}
 profile.save!
 
 Profile.where("settings->'color' = ?", "yellow")
-#=> #<ActiveRecord::Relation [#<Profile id: 1, settings: {"color"=>"yellow", "resolution"=>"1280x1024"}>]>
+#=> #<ActiveRecord::Relation
+#=>   [#<Profile id: 1, settings: {"color"=>"yellow",
+#=>                                "resolution"=>"1280x1024"}>]>
```

Comment:
The output returned by Rails is not in this format, so original format is better.